src/app/service.ts

```
@@ -0,0 +1,28 @@
+import * as aesjs from 'aes-js'
+
+import Config from '../config'
+
+/**
+ *
+ * @param plainText string to encrypt
+ *
+ * Returns encrypted text hex
+ */
+async function encryptString(plainText): Promise<string> {
+  const encryptionDecryptionKey = JSON.parse(Config.getAesEncryptionKey())
```

thinking whether it's better to move this outside the call so that it fails early if it's wrongly formed JSON and maybe add an extra assertion on it like it should be N many numbers and values > 0 and < ?